statsmodels/genmod/families/family.py

```diff
@@ -516,16 +516,16 @@ def loglike(self, endog, mu, scale=1.):
         llf = sum((`endog`*`mu`-`mu`**2/2)/`scale` - `endog`**2/(2*`scale`) - \
             (1/2.)*log(2*pi*`scale`))
         """
-        if isinstance(self.link, L.Power) and self.link.power == 1:
-            # This is just the loglikelihood for classical OLS
-            nobs2 = endog.shape[0]/2.
-            SSR = np.sum((endog-self.fitted(mu))**2, axis=0)
-            llf = -np.log(SSR) * nobs2
-            llf -= (1+np.log(np.pi/nobs2))*nobs2
-            return llf
-        else:
+        # if isinstance(self.link, L.Power) and self.link.power == 1:
```

Comment:
So this is really a hack right now because MixedGLM assumes the log-likelihood to be in the normal form. We should probably support Gaussian case though? In which case this needs to be changed if it hasn't already in an updated version? I think I'm on .7 right now.